tests/application/cms/test_views.py

```diff
@@ -727,6 +728,91 @@ def test_dept_user_should_be_able_to_edit_shared_page(test_app_client, logged_in
     assert measure_version.title == "this is the update"
 
 
+@pytest.mark.parametrize("measure_version", ["1.0", "1.1", "2.0"])
```

Comment:
It's nice to have this coverage for all version possibilities, but I'm not sure why we think behaviour might be different for different version numbers? 
Maybe just checking one in the tests would be fine... maybe a random version number picked by the Factory, which would then give us coverage of all variations still, just not all in the same test run.

Reply:
Man, I tried to write copious notes this time and still didn't make stuff clear enough. Sad times.

There's is a bit of logic in the measure version update flow that is activated only for 1.0 versions - updating the measure slug when the title changes. I've added a few different measure versions because this specific thing caught me out as I was writing this PR. For updates to 1.0 versions, it was sending two UPDATE statements to the database, taking `db_version_id` from 1 to 3 in a single 'Save' by the user. That's no longer the case, but I felt like it was still worth testing and locking this in as safely as possible.